Actions/PullRequestStatusCheck/action.yaml

```
@@ -0,0 +1,31 @@
+name: PowerShell script
+author: Microsoft Corporation
+inputs:
+  shell:
+    description: Shell in which you want to run the action (powershell or pwsh)
+    required: false
+    default: powershell
+  token:
+    description: The GitHub token running the action
+    required: false
+    default: ${{ github.token }}
+
+runs:
+  using: composite
+  steps:
+    - name: run
+      shell: ${{ inputs.shell }}
+      env:
+        GITHUB_TOKEN: ${{ inputs.token }}
```

Can't we just use `${{ github.token }}` here? What's the point of having it as an input?

```suggestion
        GITHUB_TOKEN: ${{ github.token }}
```

---

I think we can. I was under the impression that GitHub actions could only access the Github_Token if it is explicitly passed. That doesn't seem to be the case https://docs.github.com/en/actions/security-guides/automatic-token-authentication :)